microsoft-edge/webview2/release-notes.md

```diff
@@ -164,6 +164,53 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
 <!-- end of Oct 2023 Release SDK -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2164-prerelease
+
+Release Date: October 17, 2023
+
+[NuGet package for WebView2 SDK 1.0.2164-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2164-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 120.0.2164.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+* Added the `FailureSourceModulePath` property to the `ProcessFailedEventArgs` type, to specify the full path of the module that caused the crash in cases of Windows code integrity failures - that is, when a process exited with `STATUS_INVALID_IMAGE_HASH`.  See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.

```

Comment:
I added this link initially, but I think on second thought, let's remove the last line `See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.` from this. The link does not provide very helpful information, and having Intune appear here might be confusing.

Reply:
removed:
`See also [Enable code integrity](/mem/intune/user-help/you-need-to-enable-code-integrity) in the Microsoft Intune docs.`